controllers/configuration_controller.go

```diff
@@ -170,12 +210,12 @@ func terraformApply(ctx context.Context, k8sClient client.Client, namespace stri
 	return err
 }
 
-func terraformDestroy(ctx context.Context, k8sClient client.Client, namespace, configurationName, destroyJobName, tfInputConfigMapsName, applyJobName, providerName string) error {
+func terraformDestroy(ctx context.Context, k8sClient client.Client, namespace string, configuration v1beta1.Configuration, configurationName, destroyJobName, tfInputConfigMapsName, applyJobName, providerName string) error {
```

Comment:
As you can hold the Configuration object, so `configurationName` isn't needed.

Reply:
ok